src/content/faq/faq.mdx

```diff
@@ -443,22 +443,9 @@ No, it is never appropriate to use your GSA SmartPay account for expenses not re
 Yes. If you need to reach your contractor bank during the shutdown, please call the 1-800 number on the back of your GSA SmartPay charge card.
 </Accordion>
 <Accordion heading_level="h4" question="Do I still have to pay my individually billed travel account statement if I am unable to pay due to the shutdown?">
-In general, the answer is “no,” provided you are affected by the shutdown and therefore unable to pay. Payment would be required once funding is restored.
-
-In the event of a shutdown, the contractor banks recognize that affected agencies and account holders may be unable to make timely payments. Therefore, accounts could become past due until funding becomes available. For individually billed travel accounts in good standing as of the start of the shutdown, the contractor banks will:
-- Keep accounts open unless deactivation is requested by agency officials.
-- Take appropriate action to prevent accounts from becoming delinquent, suspended or canceled.
-- Ensure finance charges are not assessed.
-- Waive all applicable late payment fees.
-- Continue to generate account statements.
-- Work with agencies to maintain risk mitigation measures.
-- Coordinate with agencies and account holders to bring accounts up to date once funding becomes available.
-
-The banks use highly automated account management processes that may require manual intervention by bank staff during a shutdown. As a result, isolated instances of errant activities such as the issuance of late payment notices on an account affected by the shutdown can occur. Should this happen, please follow your agency procedures or contact your servicing contractor bank as appropriate.
+Citibank: For agencies, organizations, and card/accountholders unable to make payments in the event of a government-wide shut down, Citibank will ensure that during such periods accounts will not age delinquent or be suspended or cancelled and finance charges will not be assessed. The bank will continue to generage Statements of Account.
```

Comment:
Consider splitting `card/accountholders` to `card/account holders` (add space).

See [style guide for hyphen use](https://github.com/GSA/smartpay-website/wiki/GSA-SmartPay%C2%AE-%7C-Style-Guide-for-the-Website-and-Training-Platform#hyphens) (should be `governmentwide`).

Consider rewording `age delinquent` to a more plain language version.